python/python/glide/async_commands/sorted_set.py

```diff
@@ -169,6 +170,145 @@ def __init__(
         self.limit = limit
 
 
+class GeospatialData:
+    """
+    Represents a geographic position defined by longitude and latitude.
+
+    The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+        - Valid longitudes are from -180 to 180 degrees.
+        - Valid latitudes are from -85.05112878 to 85.05112878 degrees.
+
+    Args:
+        longitude (float): The longitude coordinate.
+        latitude (float): The latitude coordinate.
+    """
+
+    def __init__(self, longitude: float, latitude: float):
+        self.longitude = longitude
+        self.latitude = latitude
+
+
+class GeoUnit(Enum):
+    """
+    Enumeration representing distance units options for the `GEODIST` command.
+    """
+
+    METERS = "m"
+    """
+    Represents distance in meters.
+    """
+    KILOMETERS = "km"
+    """
+    Represents distance in kilometers.
+    """
+    MILES = "mi"
+    """
+    Represents distance in miles.
+    """
+    FEET = "ft"
+    """
+    Represents distance in feet.
+    """
+
+
+class OrderBy(Enum):
+    """
+    Enumeration representing sorting order options for `GEOSEARCH` command.
+    """
+
+    ASCENDING = "ASC"
+    """
+    Sort returned items from the nearest to the farthest, relative to the center point.
+    """
+    DESCENDING = "DESC"
+    """
+    Sort returned items from the farthest to the nearest, relative to the center point.
+    """
+
+
+class GeoSearchByRadius:
+    """
+    Represents search criteria of searching within a certain radius from a specified point.
+
+    Args:
+        radius (float): Radius of the search area.
+        unit (GeoUnit): Unit of the radius. See `GeoUnit`.
+    """
+
+    def __init__(self, radius: float, unit: GeoUnit):
+        """
+        Initialize the search criteria.
+        """
+        self.radius = radius
+        self.unit = unit
+
+    def __str__(self) -> str:
```

Comment:
```suggestion
    def to_args(self) -> str[]:
```

Reply:
why?

Reply:
https://github.com/aws/glide-for-redis/pull/1482#discussion_r1617880265

Reply:
I'm not sure I understand, this is exposing sensitive information?

Reply:
No, but to follow our style of converting options to arguments for other commands/clients - search for existing `to_args` and `toArgs`.
It is also a bit weird that you concatenate args into a single string and then split it.